src/components/JoinUs/Work.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import breakpoint from 'styled-components-breakpoint'
+import remcalc from 'remcalc'
+import { Link } from 'gatsby'
+import { Padding } from 'styled-components-spacing'
+import styled from 'styled-components'
+import { Row, CompensatedCol, CompensatedRow } from '../grid'
+import { H2, H3, Paragraph } from '../Typography'
+import colorLuminance from 'color-luminance'
+
+import {
+  Section,
+  FirstColumn,
+  SecondColumn,
+  Separator,
+  List,
+  TitleAndBody
+} from './elements'
+
+const MAX_CASE_STUDIES = 3
+
+/* Utility function to convert an HEX color into the RGB format */
+const hexToRgb = hex => {
+  var result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hex)
+  return result
+    ? {
+        r: parseInt(result[1], 16),
+        g: parseInt(result[2], 16),
+        b: parseInt(result[3], 16)
+      }
+    : null
+}
+
+/* Returns the luminance (in a 0-255 range) of a color in the HEX format */
+const getColorLuminance = hexColor => {
+  const rgbValue = hexToRgb(`#${hexColor}`)
+
+  return colorLuminance(rgbValue.r, rgbValue.g, rgbValue.b)
+}
+
+const CaseStudyLink = styled(Link)`
+  background: ${props => props.bg};
+  color: ${props => props.color || props.theme.colors.white};
+  flex: 1;
+`
+
+const CaseStudy = ({ children, ...props }) => (
+  <CaseStudyLink {...props}>
+    <Padding
+      vertical={{ smallPhone: 2, smallTablet: 1.5, tablet: 3 }}
+      horizontal={{ smallPhone: 2, smallTablet: 1.5, tablet: 2 }}
+    >
+      {children}
+    </Padding>
+  </CaseStudyLink>
+)
+
+const CaseStudyWrapper = styled(Padding)`
+  display: flex;
+  width: 100%;
+`
+const CaseStudyTitle = styled(H3)`
```

Comment:
This should already be one of [our Typography components](https://app.zeplin.io/project/5b4e666f0d4230a16dffd93a/screen/5bf54a64af23577ecb48d2be)

...but to be fair, in terms of the codebase, I don't actually see an appropriate one :confused: 

![image](https://user-images.githubusercontent.com/15656538/51486205-f50c4500-1d97-11e9-9ae9-9aed0462e279.png)

Reply:
Yup, not sure what to do here. The specs for the typography vs what we have are hard to match, at least for me..

Reply:
Hmm, yeah ok. Maybe worth a review tomorrow. But I reckon the right step would be for us to adjust the existing typography components to match designs, rather than make an exception for this case.

What do you think?

Reply:
I think so, yes. Let's have a quick chat before I start implementing to make sure I'll go in the right direction. (or maybe someone else wants to pick it up)

Reply:
Just mentioned this on [this trello card](https://trello.com/c/PRcbDKOg) so it's not forgotten